cmd/winch/main.go

```diff
@@ -52,12 +52,12 @@ var (
 		"server_mapper_config",
 		&pb_config.MapperConfig{},
 		"Contents of the Winch Mapper configuration. Content or read from file if _path suffix.").
-		WithFileFlag("default_winch_mapper.json").WithValidator(validateMapper)
+		WithFileFlag("").WithValidator(validateMapper)
```

Comment:
WHy is this change here?

Reply:
because it is annoying, sorry for that.
Do you really want separate PR for something that small?

Reply:
No I just did not know what was this doing and whether this change was intentional :).